server/src/callbacks.js

```diff
@@ -1,8 +1,12 @@
 import { Callbacks } from "@empirica/admin";
+import fetch from "node-fetch";
 
 const Empirica = new Callbacks();
 export default Empirica;
 
+const validLinkWithBlob = new RegExp('(https://raw.).*(blob/)(?=.*\d)(?=.*[a-z]).*(.md)');
```

Comment:
Could you give a comment above to describe what this regex should match, and an example that matches it? Same for below. =)